### ISSUE TITLE
Diagonal Gaussian distribution?

### ISSUE THREAD

**Issue by emadboctorx** [tensorflow/probability#1302]
I'm looking for the equivalent of the probability distribution implementation [here][1]. Below are the methods that I'm interested in. Is there an existing implementation that achieves the same thing?

    import numpy as np
    import tensorflow as tf
    
    
    class DiagGaussianPd:
        def __init__(self, mean, logstd):
            self.mean = mean
            self.logstd = logstd
            self.std = tf.exp(logstd)
    
        def neglogp(self, x):
            return (
                0.5 * tf.reduce_sum(tf.square((x - self.mean) / self.std), axis=-1)
                + 0.5 * np.log(2.0 * np.pi) * tf.cast(tf.shape(x)[-1], dtype=tf.float32)
                + tf.reduce_sum(self.logstd, axis=-1)
            )
    
        def entropy(self):
            return tf.reduce_sum(self.logstd + 0.5 * np.log(2.0 * np.pi * np.e), axis=-1)
    
        def sample(self):
            return self.mean + self.std * tf.random.normal(tf.shape(self.mean))

  [1]: https://github.com/openai/baselines/blob/b99a73afe37206775ac8b884d32a36e213a3fac2/baselines/common/distributions.py#L146

**Comment by brianwa84** [comment]
MultivariateNormalDiag should do it.

**Comment by emadboctorx** [comment]
@brianwa84 thanks

**Comment by emadboctorx** [comment]
@brianwa84 thanks, what should be the `loc`, `scale_diag`, `scale_identity_multiplier` parameter values be in this case?
